packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -875,7 +875,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private _disposed = false;
     public get disposed() { return this._disposed; }
 
-    private dirtyContainer = false;
+    private dirtyContainer = true;
```

Comment:
I think the intention is - constructor will / has to set right value. If so, I'd remove initialization part here and leave only declaration, to help compiler catch bugs if constructor would not set it up properly